proto/exocore/operator/v1/validator.proto

```
@@ -0,0 +1,83 @@
+syntax = "proto3";
+package exocore.operator.v1;
+
+import "amino/amino.proto";
```

buf

import "amino/amino.proto": file does not exist

(COMPILE)